copulas/__init__.py

```diff
@@ -2,7 +2,7 @@
 
 __author__ = 'DataCebo, Inc.'
 __email__ = 'info@sdv.dev'
-__version__ = '0.12.4.dev0'
+__version__ = '0.12.4.dev11'
```

Comment:
I think we should set this back to whatever will work for actual pypi

Reply:
I updated to `0.12.4.dev3`, since `0.12.4.dev2` was the most recent release:
https://pypi.org/project/copulas/#history

Reply:
you can make it 0.12.3.dev0. I yanked those dev releases

Reply:
I think you meant `0.12.4.dev0`. The `0.12.3.dev0` has not been yanked.

Reply:
nvm you're right it should be 0.12.4.dev3